### PR TITLE
Strawpoll site added

### PR DESCRIPTION
**Pull request by MattiaRaffaele**
# Please read
I had a little trouble making this request because of the mess of commits I made and I apologize for that.

# Content
In this request I've added strawpoll, a site where you can make simple polls for your services:

Strawpoll homepage
thanks for your patience

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -8382,15 +8382,6 @@
...
     },
 
-    {
-        "name": "Instacart",
-        "url": "https://www.instacart.com/user_privacy/form/",
-        "difficulty": "easy",
-        "domains": [
-            "instacart.com"
-        ]
-    },
```
Comment (tupaschoal) [inline _data/sites.json:8392]: Is the removal of instacart intentional?

Comment (MattiaRaffaele) [reply _data/sites.json:8392]: I deleted it because i saw two versions of it in visual studio, maybe it was a bug or my folder wasnt up to date

**Comment by tupaschoal** [comment]
What happened?

**Comment by MattiaRaffaele** [comment]
> What happened?

Sorry it was all a mess of commits and stuff so i deleted the entire fork and made a new one, i started working wirh branches and they make the work easier.
In these days i'll remake a pull request for strawpoll.com
Thanks for you pacience

**Comment by tupaschoal** [comment]
Ah no problem! If you need help, let us know :)

**Pull request closed without merging**